frontend/src/graph/edit-edge.jsx

```diff
@@ -57,7 +56,12 @@ export const EditEdge = ({ edge, onClose }) => {
             availableColumns = Array.from(
                 new Set(
                     successorNodeIds
-                        .filter((nodeId) => availableNodeIds.has(nodeId))
+                        .filter(
+                            (nodeId) =>
+                                availableNodeIds.has(nodeId) &&
+                                graph.getNode(nodeId).data.predecessorColumns
+                                    .length > 0
```

Comment:
cases that predecessor had not columns set up in the output, it would break the code further down in the .map, so, included a condition to only go further if there are columns to be mapped